README.md

```diff
@@ -1,55 +1,16 @@
-# MultiversX SDK for JavaScript and TypeScript: wallet components
-
-Wallet components (generation, signing) for TypeScript (JavaScript).
-
-## Distribution
-
-[npm](https://www.npmjs.com/package/@multiversx/sdk-wallet)
-
-## Installation
-
-`sdk-wallet` is delivered via [npm](https://www.npmjs.com/package/@multiversx/sdk-wallet), therefore it can be installed as follows:
-
-```
-npm install @multiversx/sdk-wallet
-```
-
-## Development
-
-Feel free to skip this section if you are not a contributor.
-
-### Additional dependencies
-
-Instanbul, for code coverage:
+# This repo was moved to [sdk-core](https://www.npmjs.com/package/@multiversx/sdk-core) and will soon be deprecated
+For the update you will need to fix the imports 
```

Comment:
> To utilize the functionality from the unified `@multiversx/sdk-core` library, please update the existing import statements within your project.

(since it's not "a mere update", it's, well, the _last update_ of the repository)

:upside_down_face: